claripy/backends/backend_concrete.py

```diff
@@ -145,6 +145,21 @@ def _solution(self, expr, v, extra_constraints=(), solver=None, model_callback=N
             raise UnsatError('concrete False constraint in extra_constraints')
         return self.convert(expr) == v
 
+    # Override Backend.is_true() for a better performance
+    def is_true(self, e, extra_constraints=(), solver=None, model_callback=None):
+        if e in {True, 1, 1.}:
+            return True
+        if type(e) is Base and e.op == "BoolV" and len(e.args) == 1 and e.args[0] is True:
+            return True
+        return super().is_true(e, extra_constraints=extra_constraints, solver=solver,model_callback=model_callback)
+    # Override Backend.is_false() for a better performance
+    def ais_false(self, e, extra_constraints=(), solver=None, model_callback=None):
```

Comment:
```suggestion
    def is_false(self, e, extra_constraints=(), solver=None, model_callback=None):
```

Probably would have been caught, but just in case...